playbooks/aws/openshift-cluster/launch.yml

```diff
@@ -25,6 +25,14 @@
       cluster: "{{ cluster_id }}"
       type: "{{ k8s_type }}"
 
+  - set_fact:
+      a_master: "{{ master_names[0] }}"
+  - add_host: name={{ a_master }} groups=service_master
+
 - include: update.yml
 
+- include: ../../common/openshift-cluster/create_services.yml
+  vars:
+     create_services__master: "{{ service_master }}"
```

Comment:
Actually, hmm, this is including another playbook, so this variable is actually a global.

So I think it should be this (I added the svc because "master" is just way too generic as a global):
```
g_svc_master: "{{ service_master }}"
```